lib/sapience/grape.rb

```diff
@@ -11,6 +11,10 @@ class API
 
 module Sapience
   class Grape
+    UnsupportedVersion = Class.new(Exception)
```

Comment:
Why are we defining the class like this... its a bit odd looking.

And shouldn't we inherit from StandardError rather than Exception